spec/es5.d.ts

```diff
@@ -0,0 +1,13 @@
+interface Symbol { }
+interface SymbolConstructor {
+  iterator: symbol;
+}
+declare var Symbol: SymbolConstructor;
+interface Iterable<T> {
+    [Symbol.iterator](): Iterator<T>;
+}
+interface Iterator<T> {
```

Comment:
`es6-shim` doesn't actually shim in `Symbol`, so the `IteratorShim<T>` doesn't have the appropriate `[Symbol.iterator]()` attached to it.